shapely/constructive.py

```diff
@@ -662,6 +663,30 @@ def polygonize_full(geometries, **kwargs):
     return lib.polygonize_full(geometries, **kwargs)
 
 
+@requires_geos("3.11.0")
+def remove_repeated_points(geometry, tolerance, **kwargs):
+    """Returns a copy of a Geometry with repeated points removed.
+
+    From the start of the coordinate sequence, each next point within the tolerance is removed.
+
+    Removing repeated points with a non-zero tolerance may result in an invalid geometry being returned.
+
+    Parameters
+    ----------
+    geometry : Geometry or array_like
+    tolerance : float or array_like
```

Comment:
Consider adding `tolerance` description from [here](https://github.com/libgeos/geos/blob/8bb8356694a57bd31a19e13d0c9ed77df7b5005e/capi/geos_c.h.in#L2868).